### ISSUE TITLE
Environmental cost of additive solutions

### ISSUE THREAD

**Issue by joedolson** [karlgroves/overlayfactsheet#6]
Employing an overlay rather than addressing the underlying problems requires unnecessary extra downloads on every page load in additional to unnecessary processing costs. See: https://www.aceee.org/files/proceedings/2012/data/papers/0193-000409.pdf. At this time, the utilization cost of 1GB of data was estimated at 5kWh.

Due to the character of an overlay, all the energy costs of the original site remain intact, and the overlay is an added environmental cost.

While some energy cost is inevitable in any website, the costs associated with overlays are unnecessary, as the underlying problems can be solved without the added load.

**Comment by karlgroves** [comment]
Would this warrant its own section on the fact sheet?

**Comment by stringyland** [comment]
I wonder if it might be worth including as part of performance and/or data problems? More to download equals slower performance, higher data charges for many, and increases the energy cost.

**Comment by joedolson** [comment]
Agreed; it's an overall part of performance. Noting that the higher data charges & impact on devices can be an issue for areas of low connectivity and internet service, as well, e.g. rural networks.

**Comment by karlgroves** [comment]
@joedolson Can you take a stab at writing something up for this?

**Comment by DagA11y** [comment]
We could maybe build a simple proof of concept site and check it against https://www.websitecarbon.com/
Then we could have subdomains for it with different overlays (same site but with overlay(s)) and then measure it again against https://www.websitecarbon.com/ and get the facts...

It would be quite adequate I guess. We would just have to buy the darn thing(s)...

**Comment by joedolson** [comment]
Yes, that occurred to me, too. But actually having to set up accounts and acquire all of these tools is a distasteful and expensive proposition...

I think we'd also want to run some performance tests on the client side, to see the impact of overlays on processor use, etc. 

That is something that would give us some fairly solid data without needing to do all the research on the broader environmental implications of poor performance, since we'd have some direct correlations. Maybe we should do a cost estimate and see if some organization might sponsor it.
